File: Files/task1.py
#
#  task1.py
#
#
#  Created by Angelos Stavrou

# generate random integer values
from random import seed
from random import getrandbits
import time
# seed random number generator
print(time.time())
#seed(time.time())
# generate some integers
value = [0] * 16
for index in range(16):
   value[index] = getrandbits(8)

print ('[{}]'.format(', '.join(hex(x) for x in value)))

File: Files/task1.c
//
//  task1.c
//  
//
//  Created by Angelos Stavrou
//

#include <stdio.h>
#include <stdlib.h>
#include <time.h>

#define KEYSIZE 16

int main(void)
{
  int i;
  char key[KEYSIZE];

  printf("%lld\n", (long long) time(NULL));
  //srand (time(NULL));      // This is where srand() is called using time

  for (i = 0; i< KEYSIZE; i++){
     key[i] = rand()%256;
     printf("%.2x", (unsigned char)key[i]);
  }
  printf("\n");
  return 0;
}
